certified-connectors/LMS365/apiDefinition.swagger.json

```diff
@@ -4086,6 +4098,7 @@
                 }
               },
               "required": [
+                "Title",
```

Comment:
Hello @elearningforce-db,
The required input field "Title" has been added. Existing flows will not send this new field.
Can you please confirm that this change will not affect existing users?
Please let me know if you have any questions.

Reply:
Without this field, existing flows already does not work as the API was changed and "Title" is required.
So the PR is some sort of hot-fix basically.
So I'm confirming it is a correct way.